### PR TITLE
create recognition component

### PR DESCRIPTION
**Pull request by jitendrasbunde**

### CODE REVIEW

File: react-frontend/src/components/createRecognitionComponent.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
+import React, { useState, useReducer, useEffect } from "react";
+import { Form, Button, Col, Row, Card, InputGroup } from "react-bootstrap";
+import reducer from "components/reducers/createRecognitionReducer";
+import { setDetails } from "components/actions/createRecognitionAction";
+import { string, object } from "yup";
+import PropTypes from "prop-types";
```
Comment (sahilbhatia) [inline react-frontend/src/components/createRecognitionComponent.js:6]: @jitendrasbunde , @mayuriardad 

[React Development Guidelines](https://github.com/joshsoftware/peerly/wiki/React-Development-Guidelines) not met.

File: react-frontend/src/components/createRecognitionComponent.js [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+};
+
+const CreateRecognition = (props) => {
+  const {
+    userPlaceholder, // eslint-disable-line react/prop-types
+    users, // eslint-disable-line react/prop-types
+    coreValue, // eslint-disable-line react/prop-types
+    textareaPlaceholder, // eslint-disable-line react/prop-types
+    coreValuePlaceholder, // eslint-disable-line react/prop-types
+  } = props; // eslint-disable-line react/prop-types
```
Comment (sahilbhatia) [inline react-frontend/src/components/createRecognitionComponent.js:21]: @jitendrasbunde , @mayuriardad 

[React Development Guidelines](https://github.com/joshsoftware/peerly/wiki/React-Development-Guidelines) not met.

File: react-frontend/src/components/createRecognitionComponent.js [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  const [state, dispatch] = useReducer(reducer, initialState);
+  const [isLoading, setIsLoading] = useState(false);
+  const [validated, setValidated] = useState(false);
+  const [labelData, setLabelData] = useState("");
+
+  useEffect(() => {
+    console.log("Loading the page  " + isLoading); // eslint-disable-line no-console
```
Comment (sahilbhatia) [inline react-frontend/src/components/createRecognitionComponent.js:28]: @jitendrasbunde , @mayuriardad 

[React Development Guidelines](https://github.com/joshsoftware/peerly/wiki/React-Development-Guidelines) not met.

**Comment by jitendrasbunde** [comment]
This PR not for merge

**Pull request closed without merging**